zebra-chain/src/orchard/tree.rs

```diff
@@ -153,9 +155,52 @@ impl ZcashDeserialize for Root {
 }
 
 /// A node of the Orchard Incremental Note Commitment Tree.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Copy, PartialEq)]
```

Comment:
We normally derive Eq where we can.

```suggestion
#[derive(Copy, Clone, Debug, Eq, PartialEq)]
```